segmentation/include/pcl/segmentation/impl/lccp_segmentation.hpp

```diff
@@ -285,6 +285,24 @@ pcl::LCCPSegmentation<PointT>::segment (std::map<uint32_t, typename pcl::Supervo
   // Correct edge relations using extended convexity definition if k>0
   applyKconvexity (k_factor_);
 
+  // Determine wether to use cutting planes
+  doGrouping ();
+  grouping_data_valid_ = true;
+}
+
+template <typename PointT> void
+pcl::LCCPSegmentation<PointT>::doGrouping ()
+{
+  // clear the processed_ map
+  seg_label_to_sv_list_map_.clear ();
+  for (typename std::map<uint32_t, typename pcl::Supervoxel<PointT>::Ptr>::iterator svlabel_itr = sv_label_to_supervoxel_map_.begin ();
+      svlabel_itr != sv_label_to_supervoxel_map_.end (); ++svlabel_itr)
+  {
+    const uint32_t sv_label = svlabel_itr->first;
```

Comment:
I think that const reference is treated as an "alias" by the compiler, and therefore incurs no runtime penalty. A const variable is not just an "alias", it actually has memory allocated for it and hence an address. So creating such will incur copying. Unless (as in a trivial case like ours) it is optimized.

But I have to admit that these are just my speculations. I wasn't able to google any proof. The only relevant discussion is [this](http://stackoverflow.com/questions/2641907/do-variable-references-alias-incure-runtime-cost), however the references are not const in that case.

So... do as you prefer, it's a tiny issue anyway.